subsys/random/Kconfig

```diff
@@ -31,6 +31,7 @@ config TEST_RANDOM_GENERATOR
 config TIMER_RANDOM_INITIAL_STATE
```

Comment:
Just out of logical grouping, could we also move this right under `TIMER_RANDOM_GENERATOR`?

Reply:
If it was placed under `TIMER_RANDOM_GENERATOR` is would form part of the `choice RNG_GENERATOR_CHOICE` group

Reply:
Right. Though it wouldn't if it were enclosed by an `if` (I think). Or then it could be moved right after the `endchoice`. Or also just left as is...